k2/csrc/fsa_algo.cc

```diff
@@ -50,6 +52,41 @@ inline int32_t InsertIntersectionState(
   return result.first->second;
 }
 
+static void TraceBackRmEpsilonLogSum(
```

Comment:
Compare this with how carefully I document functions like this in determinize.cc.

Reply:
OK, will add in another PR.